### ISSUE TITLE
Blog: Prometheus

### ISSUE THREAD

**Issue by philwinder** [microservices-demo/microservices-demo#482]
Write another blog post (as a part of a series with the other blog posts) about prometheus and the integration with sock shop. More focus should be placed on sock shop in this blog.

Link to old post: http://container-solutions.com/microservice-monitoring-with-prometheus/

**Comment by pidster** [comment]
Blog series:
- [ ] Introductory blog for Prometheus basics with Sock Shop
- [ ] Prometheus + go-lang 
- [ ] Prometheus + Node/Javascript
- [ ] Prometheus + Java/Spring

**Comment by pidster** [comment]
Research & notes needed for each language type, investigate whether names should be the same for all similar metrics (e.g. http metrics in go, node & spring)

**Comment by pidster** [comment]
Target publishing date is April, but the blogs need to be ready for review at least week beforehand.

**Comment by vlal-zz** [comment]
@pidster I don't see any reference to Cortex or Weave Cloud in the above. Should I include any mention of it or will that come from your side?

**Comment by pidster** [comment]
@vlal the gist of this blog series is intended to be from the point of view of the developer, instrumenting their application. You can state that we actually send the data to Weave Cloud.

We'll write about how we use the data, separately.

**Comment by lukemarsden** [comment]
I'd be equally happy for this to be a writeup of the work that @philwinder and @nustiueudinastea are doing now.

**Comment by philwinder** [comment]
Yes. This is basically my presentation. Basically an extension of http://container-solutions.com/microservice-monitoring-with-prometheus/

**Comment by lukemarsden** [comment]
I think we agreed that @vlal was just gonna finish the blog post he's started in the end, because he's already almost finished it.

**Comment by vlal-zz** [comment]
@lukemarsden do you still require the other blogs posts in the series?

**Comment by lukemarsden** [comment]
Let's hold on the other blog posts until we get a chance to discuss them. Let's just get the first one out (that I believe you've already started on).

**Comment by vlal-zz** [comment]
@lukemarsden done
https://docs.google.com/document/d/1TJacdvylGI5TrPvSncUSswAdASgmf6i_snFdsXUlY98/edit

**Comment by jasonrichardsmith** [comment]
@mewzherder  another blog post
